File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    let result = '';
    let index = 0;
    for (let i = 0; i < s.length; i++) {
        if (s[i] === " ") {
            index = 0;
            result += " ";
            continue;
        }
        if (index % 2 === 0) {
            result += s[i].toUpperCase();
        } else {
            result += s[i].toLowerCase();
        }
        index++;
    }
    return result
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    result = []
    for word in s.split(" "):
        temp = ''
        for j in range(len(word)):
            if j % 2 == 0: 
                temp += word[j].upper()
            else:
                temp += word[j].lower()
        result.append(temp)
        
    return " ".join(result)